src/app_service/service/provider/async_tasks.nim

```diff
@@ -11,13 +11,10 @@ const postMessageTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
 
   let jsonMessage = arg.message.parseJson
```

Comment:
do u need to parse it? you can just pass the message to the provider request

Reply:
changed :)